lib/oli_web/views/layout_view.ex

```diff
@@ -131,4 +132,58 @@ defmodule OliWeb.LayoutView do
   defp show_collab_space?(nil), do: false
   defp show_collab_space?(%CollabSpaceConfig{status: :disabled}), do: false
   defp show_collab_space?(_), do: true
+
+  def render_license(%{license: %{license_type: :custom} = license} = assigns) do
+    assigns = Map.put(assigns, :custom_license_details, license.custom_license_details)
+
+    ~H"""
+    <div
+      id="license"
+      class="container mx-auto flex flex-row px-10 items-center overflow-y-auto h-[40px] max-h-[40px] relative top-0"
+    >
+      <p class="h-full">License: <%= @custom_license_details %></p>
+    </div>
+    """
+  end
```

Comment:
I think we can avoid assigning a new variable and directly access the custom license details from the license struct
```suggestion
  def render_license(%{license: %{license_type: :custom}} = assigns) do
    ~H"""
    <div
      id="license"
      class="container mx-auto flex flex-row px-10 items-center overflow-y-auto h-[40px] max-h-[40px] relative top-0"
    >
      <p class="h-full">License: <%= @license.custom_license_details %></p>
    </div>
    """
  end
```

Reply:
Thanks for that suggestion! I put it like you said and it's working fine. For some reason I added the Map.put because I got a warning but it's not happening anymore. 🎉